### ISSUE TITLE
Android: tethering

### ISSUE THREAD

**Issue by davidpanderson** [BOINC/boinc#1568]
I got a user email saying the BOINC doesn't support bluetooth tethering (i.e. routing IP traffic over bluetooth).  Is this an app-specific function?  Doesn't seem like it would be.

**Comment by GITNE** [comment]
> Is this an app-specific function?

You are correct, this is not an app-specific function, at least generally speaking. However, a badly written app can break this feature. It depends on what IP address an app binds to for outgoing connections. If it binds to ```0.0.0.0``` — as it should — then the IP stack should not only be able to figure out the route to the destination IP address but also which local NIC (in this case a tethered Bluetooth device) to use automatically. If the app binds to a specific local IP address of a NIC which does not happen to be a tethered Bluetooth device then obviously tethering over Bluetooth will not work.

I think that perhaps the user did not setup the router or NAT at the device tethered via Bluetooth. Tethering is not some sort of automagical technology. It still requires a working proxy router or NAT (in most cases).
